### ISSUE TITLE
Can you explain detectScreenOrientation? Add detectScreenOrientation

### ISSUE THREAD

**Issue by psyhonut** [Valve/fingerprintjs2#61]
Hi!
Can you explain, why you do this and how to detect screen orientation, if you sort available resolutions? 
```javascript
       if(this.options.detectScreenOrientation) {
          available = (screen.availHeight > screen.availWidth) ? [screen.availHeight, screen.availWidth] : [screen.availWidth, screen.availHeight];
        } else {
          available = [screen.availHeight, screen.availWidth];
        }
```

And maybe you can add separate method to detect device orientation availability? (but i dunno how to detect it, maybe you know :-))

**Comment by Valve** [comment]
Hi, @psyhonut . I don't see where I sort it, could you clarify.

THis code just looks if screen width is less than height, which means the screen was rotated 90 degrees.
If it was, we switch height with width to make it rotation independent.

**Comment by psyhonut** [comment]
```
? [screen.availHeight, screen.availWidth] : [screen.availWidth, screen.availHeight]
````
Why did you do this? I cant understand. Can you explain?

**Comment by Valve** [comment]
let's say your screen orientation is normal (landscape), and your screen has this resolution: 1920x1080.
This means width = 1920 and height = 1080. In this case we'll end up with [1920, 1080] array for resolution.
Then tomorrow, the person rotates the screen to 90 degrees (to read a lengthy Word document) and his resolution becomes [1080, 1920] and the fingerprint changes. 

To avoid this, we first detect the rotation and change width with height if detected. 
So if the person rotated the screen, we'll still have [1920, 1080] in the resolution.

Please let me know if you need more info :+1:

**Comment by leoshaw** [comment]
Hi, great library.  Just wanted to confirm: this option is switched off by default?  Could be worth documenting it on the wiki too - we're seeing a few cases where the fingerprint changes lots of times over the course of a session and we're wondering if it could be the user rotating the device, in which case turning this on could solve our problem.

**Comment by Valve** [comment]
@leoshaw you're right, I'll make it "on" by default, because on mobile it makes sense to have it "on" all the time. Released 1.0.0-rc2 with this change. Please have a look.

**Comment by leoshaw** [comment]
Looks good, thank you!
